config/confighttp/metrics.go

```diff
@@ -0,0 +1,82 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package confighttp
+
+import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+const (
+	metricPrefix = "otelcol_http_server"
+
+	defaultMaxHttpConnectionsMetricName = metricPrefix + "http_max_connections"
+	defaultHttpConnectionsMetricName    = metricPrefix + "http_connections"
+)
+
+// HttpServerTelemetryConfig defines the telemetry configuration for HTTPServerSettings
+type HttpServerTelemetryConfig struct {
+	// Enabled is set to enable opencensus metrics
+	Enabled bool `mapstructure:"enabled"`
+	// ExtraAttributes is the extra attributes for metrics
+	ExtraAttributes map[string]string `mapstructure:"extra_attributes"`
```

Comment:
Why do you need this parameter? Collector provides a way to add extra resource attributes to the emitted metrics in the `service::telemetry::resource` config.